### ISSUE TITLE
Change v-modal of a v-radio has no effect

### ISSUE THREAD

**Issue by zhanghaowx** [johnleider/vue-materials#24]
I am not sure if it is a limitation from vue.js or a bug, but here it is:
```
    <div class="input-field" v-for="(controller, i) in controllers">
        <v-radio name="controllers" v-model="controllerIndex" :id="'controller'+i" :val="i">{{ controller.name }}</v-radio>
    </div>
```

If I change value of `controllerIndex` later, I can see value of v-radio has changed as expected but the checked status doesn't change.

[![Capture.png](https://s11.postimg.org/eorxptkxf/Capture.png)](https://postimg.org/image/xtv6zkzlb/)

Also a small but maybe worth noting thing, I have to convert my number to a string in the v-model field, otherwise it also doesn't work.

**Comment by johnleider** [comment]
I'll take a look, ty.

**Comment by johnleider** [comment]
This is fixed with commit [0fa5b4ac8bd7a515c1d0f646726580feb158f758](https://github.com/johnleider/vue-materials/commit/0fa5b4ac8bd7a515c1d0f646726580feb158f758). Go ahead and pull vue-materials@1.0.3-rc.4 and you should be good to go.

**Comment by zhanghaowx** [comment]
Verified work. Thanks for the quick fix!
